tests/bell.py

```diff
@@ -90,7 +93,129 @@ def logout(driver):
     # wait 5 secs
     sleep(5)
 
+def goto(driver, page):
+    """ string -> NoneType
+
+    Goes to the provided page (use "Home" to go to the homepage)
+    """
+    if page == "Home":
+        driver.get(get_url())
+        sleep(5)
+        return
+    elif driver.current_url == get_url():
+        login(driver, "admin", "password")
+    link = driver.find_element_by_link_text(page)
+    link.click()
+    sleep(5)
+
+def add_member(driver, first_name = "", last_name = "", middle_name = "",
+               login = "", password = "", phone = "", email = "", language = "",
+               birthdate = None, gender = "", level = ""):
+    """ object, str, str, str, str, str, str, str, str, 
+    list[str, str, str], str, str -> tuple(str, str)
+
+    Goes to homepage, fills in new member form, and returns a tuple of strings
+    representing username and password of the new member or, if 
+    the member already exists, return a tuple of empty strings.
+    TODO: Needs testing AND refactoring!
+    """
+    birthdate = birthdate or []
```

Comment:
I had erroneously commented saying that this line is a bug.
It is not a bug, but it can be confusing... wouldn't it be better to just change the conditional below to `if bithdate == None:`?